xl-platform/xlr-ecommerce.yaml

```diff
@@ -0,0 +1,288 @@
+apiVersion: xl-release/v1beta1
```

Comment:
I still don't find this nice to be here if really needed then at least it should be part of the environment setup yaml